tools/nemo_forced_aligner/utils/data_prep.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+
+import soundfile as sf
```

Comment:
Should this be in requirements ? Like pysoundfile or something

Reply:
Won't it be downloaded during the nemo install? https://github.com/NVIDIA/NeMo/blob/main/requirements/requirements_asr.txt#L18

Reply:
Yes, I made a new environment and installed nemo_toolkit[asr] and did not try to install soundfile, and ran align.py. Worked fine - marking this as resolved.